version.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+VERSION=$(
+  cat package.json | 
+  grep version | 
+  head -1 |                 
+  awk -F: '{ print $2 }' |  # Split by ":" and get the second part
+  sed 's/[",]//g' |         # Remove quotes and commas
+  tr -d '[[:space:]]'       # Remove spaces
+)
+
+echo "version=$VERSION"
```

Comment:
Nice script! Then I remembered... what about `jq ".version" package.json`?

Reply:
You're right, thanks!